### ISSUE TITLE
The examples given in the README.md are different than actual

### ISSUE THREAD

**Issue by nishikeshKardak** [mozilla/moz-sql-parser#77]
### Describe the bug
I have tried examples given in the  README.md and I noticed that the given output is different than actual.

### Screenshots
Usage 1:
```
>>> from moz_sql_parser import parse
>>> import json
>>> json.dumps(parse("select count(1) from jobs"))
'{"from": "jobs", "select": {"value": {"count": {"literal": 1}}}}'
```
Actual : 

![Capture1](https://user-images.githubusercontent.com/39300717/54875451-9f025f00-4e25-11e9-903e-467b87e8a1e5.JPG)

Usage 2
```
>>> json.dumps(parse("select a as hello, b as world from jobs"))
'{"from": "jobs", "select": [{"name": "hello", "value": "a"}, {"name": "world", "value": "b"}]}'
```
Actual : 
![Capture2](https://user-images.githubusercontent.com/39300717/54875462-d6710b80-4e25-11e9-9793-6e8f5a951917.JPG)

Usage 3
```
# SELECT * FROM dual WHERE a>b ORDER BY a+b
{
    "select": "*",
    "from": "dual"
    "where": {"gt": ["a","b"]},
    "orderby": {"add": ["a", "b"]}
}
```
Actual:
![Capture3](https://user-images.githubusercontent.com/39300717/54875485-47182800-4e26-11e9-899f-533257a370a5.JPG)

**Comment by klahnakoski** [comment]
Thank you for pointing this out.

**Comment by nishikeshKardak** [comment]
Hello @klahnakoski, can I work on this issue??

**Comment by nishikeshKardak** [comment]
This issue fixed by #81
